### DISCUSSION TITLE
0.0.3 -- sleeping giant

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#1038] [Releases]
## What's Changed -- a lot.

![](https://media.giphy.com/media/5RxNQCK01NdAc/giphy.gif)

* Chaging CI/CD variables, checking if everything is ok by @yangwao in https://github.com/kodadot/nft-gallery/pull/467
* Flex on preloader by @yangwao in https://github.com/kodadot/nft-gallery/pull/476
* pagination has native pink by @yangwao in https://github.com/kodadot/nft-gallery/pull/481
* Tuned new profiles, with on-chain properties :) by @yangwao in https://github.com/kodadot/nft-gallery/pull/485
* fixed sharing button for collections by @yangwao in https://github.com/kodadot/nft-gallery/pull/487
* Fixing technical dept against runtime update by @vikiival in https://github.com/kodadot/nft-gallery/pull/492
* Rendering profiles and collections with base minimum on social medias by @yangwao in https://github.com/kodadot/nft-gallery/pull/496
* 8k6: Subsocial and PDF support by @yangwao in https://github.com/kodadot/nft-gallery/pull/499
* Fixed preview on collections by @yangwao in https://github.com/kodadot/nft-gallery/pull/501
* Showing user profile on social medias by @yangwao in https://github.com/kodadot/nft-gallery/pull/503
* Pinky shadows on profile by @yangwao in https://github.com/kodadot/nft-gallery/pull/515
* Profile has watchers so higher chance it will load on-chain stuff by @yangwao in https://github.com/kodadot/nft-gallery/pull/517
* Bring back classic mint by @yangwao in https://github.com/kodadot/nft-gallery/pull/520
* showing created, collected and sold by @yangwao in https://github.com/kodadot/nft-gallery/pull/523
* Ratio on full screen, sharing button fix, user profile identity  by @yangwao in https://github.com/kodadot/nft-gallery/pull/534
* Removed unused Textile by @yangwao in https://github.com/kodadot/nft-gallery/pull/536
* adding basic XSS prevention by @yangwao in https://github.com/kodadot/nft-gallery/pull/539
* add SECURITY.md and rebuild new pinata keys by @yangwao in https://github.com/kodadot/nft-gallery/pull/541
* New Artists spotlight by @yangwao in https://github.com/kodadot/nft-gallery/pull/543
* Rank -> Score, Showing twitter next to profile at nft item detail by @yangwao in https://github.com/kodadot/nft-gallery/pull/549
* Faster loads thanks to prefetch, Issuer & Owner to nft detail, don't show twitter until resolved by @yangwao in https://github.com/kodadot/nft-gallery/pull/557
* prefetch should be 3s per page now by @yangwao in https://github.com/kodadot/nft-gallery/pull/559
* Svg logo, new loader, improved seo by @yangwao in https://github.com/kodadot/nft-gallery/pull/562
* endpoint change for subsocial by @yangwao in https://github.com/kodadot/nft-gallery/pull/566
* just before 9k mountain peak by @yangwao in https://github.com/kodadot/nft-gallery/pull/572
* 9K by @vikiival in https://github.com/kodadot/nft-gallery/pull/578
* keys keys keys by @yangwao in https://github.com/kodadot/nft-gallery/pull/582
* Fixes emoji picker bug by @vikiival in https://github.com/kodadot/nft-gallery/pull/591
* Changelog: Slovenia is great by @yangwao in https://github.com/kodadot/nft-gallery/pull/607
* Fixed price chart responsivity  by @yangwao in https://github.com/kodadot/nft-gallery/pull/621
* Smol updates, fog is still low by @yangwao in https://github.com/kodadot/nft-gallery/pull/627
* Security push by @yangwao in https://github.com/kodadot/nft-gallery/pull/635
* Fixing slate API! by @vikiival in https://github.com/kodadot/nft-gallery/pull/638
* What a sunny day! 🌤 by @yangwao in https://github.com/kodadot/nft-gallery/pull/643
* Knocked Slate, better to get stable minting process by @yangwao in https://github.com/kodadot/nft-gallery/pull/650
* Sometimes after fog, comes sunny day. by @yangwao in https://github.com/kodadot/nft-gallery/pull/665
* New TS(i) 4.4 by @yangwao in https://github.com/kodadot/nft-gallery/pull/668
* Now you are fluent in all lingos by @yangwao in https://github.com/kodadot/nft-gallery/pull/669
* Viki said "v pohode" by @yangwao in https://github.com/kodadot/nft-gallery/pull/671
* Unlist, performance, transfer, moving assets to public by @yangwao in https://github.com/kodadot/nft-gallery/pull/681
* Fixed scroll from gallery to item by @yangwao in https://github.com/kodadot/nft-gallery/pull/687
* Bugs here and there by @yangwao in https://github.com/kodadot/nft-gallery/pull/696
* no NSFW check by @vikiival in https://github.com/kodadot/nft-gallery/pull/697
* Transfer from kodadot and not showing burned NFTs by @yangwao in https://github.com/kodadot/nft-gallery/pull/698
* Adding PermaFrost redirect do Discord by @yangwao in https://github.com/kodadot/nft-gallery/pull/699
* 🥶 by @yangwao in https://github.com/kodadot/nft-gallery/pull/709
* vite frost by @yangwao in https://github.com/kodadot/nft-gallery/pull/711
* Update PermaMint.vue by @vikiival in https://github.com/kodadot/nft-gallery/pull/712
* FML slash by @vikiival in https://github.com/kodadot/nft-gallery/pull/714
* Preview for Permafrost/Arweave images by @vikiival in https://github.com/kodadot/nft-gallery/pull/715
* Ice Ice baby by @vikiival in https://github.com/kodadot/nft-gallery/pull/727
* Status signalling for collections v1 goes out!  by @yangwao in https://github.com/kodadot/nft-gallery/pull/728
* Paste from clipboard to mint & USD at transfer window by @yangwao in https://github.com/kodadot/nft-gallery/pull/732
* Missing text to simple mint that we support paste from clipboard by @yangwao in https://github.com/kodadot/nft-gallery/pull/736
* Collection image by @yangwao in https://github.com/kodadot/nft-gallery/pull/742
* let's roll - preloader placeholder by @yangwao in https://github.com/kodadot/nft-gallery/pull/743
* out of control by @yangwao in https://github.com/kodadot/nft-gallery/pull/761
* Polkadot v6 into prod. by @vikiival in https://github.com/kodadot/nft-gallery/pull/764
* Wao said that 293 is my new favourite number by @vikiival in https://github.com/kodadot/nft-gallery/pull/770
* In less than 24 hrs someone can change indexer schema by @vikiival in https://github.com/kodadot/nft-gallery/pull/772
* Tabs change, donation button, mailto by @yangwao in https://github.com/kodadot/nft-gallery/pull/779
* Sharing button & icon alignment in item detail!  by @yangwao in https://github.com/kodadot/nft-gallery/pull/787
* 3D is working, added AR, optimized price chart bundle, markdown enabled for collections by @yangwao in https://github.com/kodadot/nft-gallery/pull/794
* websharing api, unified collection, historychart fix by @yangwao in https://github.com/kodadot/nft-gallery/pull/800
* Fixed one by one for images by @vikiival in https://github.com/kodadot/nft-gallery/pull/810
* Špice mušt flow by @yangwao in https://github.com/kodadot/nft-gallery/pull/818
* Einstellung effect. by @yangwao in https://github.com/kodadot/nft-gallery/pull/829
* Double fix 🎓  by @vikiival in https://github.com/kodadot/nft-gallery/pull/846
* Basic Image component, tooltip score, emoji on gallery, pagination on spotlight  by @yangwao in https://github.com/kodadot/nft-gallery/pull/866
* fix(spotlight): top pagination by @roiLeo in https://github.com/kodadot/nft-gallery/pull/868
* Are these numbers correct? by @vikiival in https://github.com/kodadot/nft-gallery/pull/867
* Are numbers all right now? by @yangwao in https://github.com/kodadot/nft-gallery/pull/869
* syncback by @yangwao in https://github.com/kodadot/nft-gallery/pull/870
* fix(EmotionList): remove spacing before title by @roiLeo in https://github.com/kodadot/nft-gallery/pull/874
* Deduplicate horizontal line at actions component #812 by @akshajb in https://github.com/kodadot/nft-gallery/pull/873
* feat(collection): filter & search by @roiLeo in https://github.com/kodadot/nft-gallery/pull/875
* fix(series): volume & days by @roiLeo in https://github.com/kodadot/nft-gallery/pull/848
* My cup of mokka by @vikiival in https://github.com/kodadot/nft-gallery/pull/773
* feat(profile): show searchBar on Created tab by @roiLeo in https://github.com/kodadot/nft-gallery/pull/890
* 888 commented subsocial impl by @vikiival in https://github.com/kodadot/nft-gallery/pull/892
* fix(gallery): top pagination align by @roiLeo in https://github.com/kodadot/nft-gallery/pull/895
* fix(series): sort query param by @roiLeo in https://github.com/kodadot/nft-gallery/pull/893
* fix(galleryItem): hotfix events actions by @roiLeo in https://github.com/kodadot/nft-gallery/pull/904
* fix(galleryItem): hotfix description line breaks by @roiLeo in https://github.com/kodadot/nft-gallery/pull/906
* feat(spotlight): show total volume column by @roiLeo in https://github.com/kodadot/nft-gallery/pull/913
* feat(profile): identicon by @roiLeo in https://github.com/kodadot/nft-gallery/pull/918
* feat(landing): discord widget by @roiLeo in https://github.com/kodadot/nft-gallery/pull/922
* fix(spotlight): display nft currentOwner by @roiLeo in https://github.com/kodadot/nft-gallery/pull/920
* fix(profile): column responsive by @roiLeo in https://github.com/kodadot/nft-gallery/pull/921
* feat(galleryItem): image orientation by @roiLeo in https://github.com/kodadot/nft-gallery/pull/916
* #745 Add motion effect on gallery cards in profile by @prachi00 in https://github.com/kodadot/nft-gallery/pull/819
* fix(galleryItem): removeProperty transform by @roiLeo in https://github.com/kodadot/nft-gallery/pull/926
* feat(collection&profile): donationButton under sharing by @roiLeo in https://github.com/kodadot/nft-gallery/pull/930
* Disabling all actions by @vikiival in https://github.com/kodadot/nft-gallery/pull/940
* Reenable buy by @vikiival in https://github.com/kodadot/nft-gallery/pull/945
* fix(searchBarCollection): remove filter toggle for now & align content by @roiLeo in https://github.com/kodadot/nft-gallery/pull/943
* feat(GalleryCard): change column layout by @roiLeo in https://github.com/kodadot/nft-gallery/pull/929
* fix(metadata): series & spotlight meta image by @roiLeo in https://github.com/kodadot/nft-gallery/pull/944
* fix(faq): responsive mobile by @roiLeo in https://github.com/kodadot/nft-gallery/pull/948
* fix(locale): show user locale flag in navbar by @roiLeo in https://github.com/kodadot/nft-gallery/pull/949
* [ImgBot] Optimize images by @imgbot in https://github.com/kodadot/nft-gallery/pull/953
* Air(Drops) comming by @vikiival in https://github.com/kodadot/nft-gallery/pull/887
* Mass OPS by @vikiival in https://github.com/kodadot/nft-gallery/pull/960
* Smol fee for the expert mint by @vikiival in https://github.com/kodadot/nft-gallery/pull/968
* Identity form by @vikiival in https://github.com/kodadot/nft-gallery/pull/970
* fix(page): layout by @roiLeo in https://github.com/kodadot/nft-gallery/pull/976
* Fixing stats for the collection by @vikiival in https://github.com/kodadot/nft-gallery/pull/983
* Mobile menu not opening/closing well #932 by @TK-Development in https://github.com/kodadot/nft-gallery/pull/993
* Minting success message has correct color by @vikiival in https://github.com/kodadot/nft-gallery/pull/1001
* fix(spotlight): layout by @roiLeo in https://github.com/kodadot/nft-gallery/pull/1002
* fix(toggleLayout): default value incorect by @roiLeo in https://github.com/kodadot/nft-gallery/pull/1003
* Added postfix to the classic mint by @vikiival in https://github.com/kodadot/nft-gallery/pull/998
* 680 glb poster by @vikiival in https://github.com/kodadot/nft-gallery/pull/1022
* Add Sub.id into profile by @fairywellos in https://github.com/kodadot/nft-gallery/pull/1006
* fix(sharing): active by @roiLeo in https://github.com/kodadot/nft-gallery/pull/1026
* Airdrop two by @vikiival in https://github.com/kodadot/nft-gallery/pull/1021
* feat(collection): pagination by @roiLeo in https://github.com/kodadot/nft-gallery/pull/1020
* Add Collection view by @TK-Development in https://github.com/kodadot/nft-gallery/pull/1019

## New Contributors
* @akshajb made their first contribution in https://github.com/kodadot/nft-gallery/pull/873
* @TK-Development made their first contribution in https://github.com/kodadot/nft-gallery/pull/993
* @fairywellos made their first contribution in https://github.com/kodadot/nft-gallery/pull/1006

**Full Changelog**: https://github.com/kodadot/nft-gallery/compare/0.0.1...0.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/kodadot/nft-gallery/releases/tag/0.0.3'>0.0.3 -- sleeping giant</a>.</em>

**Comment by vikiival** [comment]
![snorlax](https://user-images.githubusercontent.com/22471030/140504277-84f69eab-3379-449c-8c8c-d351bed8b071.jpg)
